config/webpack.target.browser.js

```diff
@@ -2,41 +2,20 @@
 
 const path = require('path')
 const webpack = require('webpack')
-const HtmlWebpackPlugin = require('html-webpack-plugin')
+const merge = require('webpack-merge')
+const appConfig = require('./webpack.target.app')
 
-module.exports = {
-  entry: [
-    require.resolve('@babel/polyfill'),
-    path.resolve(__dirname, '../src/main')
-  ],
+module.exports = merge(appConfig, {
+  resolve: {
+    extensions: ['.browser.js', '.browser.jsx']
+  },
```

Comment:
it can be a great addition to cc default config I think. 

I think it's too bad that banks doesn't use https://github.com/cozy/create-cozy-app/blob/master/packages/cozy-scripts/config/webpack.target.browser.js & https://github.com/cozy/create-cozy-app/blob/master/packages/cozy-scripts/config/webpack.target.mobile.js

Reply:
> I think it's too bad that banks doesn't use cozy/create-cozy-app:packages/cozy-scripts/config/webpack.target.browser.js@master & cozy/create-cozy-app:packages/cozy-scripts/config/webpack.target.mobile.js@master 


Yes we cannot solve all problems in one go. Let's merge this first.